MapleServer2/Commands/Game/DebugCommands.cs

```diff
@@ -0,0 +1,32 @@
+﻿using MapleServer2.Commands.Core;
+namespace MapleServer2.Commands.Game;
+
+public class DungeonTests : InGameCommand
```

Comment:
debug stuff?

Reply:
yea but it will be needed for future testing / more dungeon stuff implementation testing